recipes/cppyy-cling/build.sh

```diff
@@ -0,0 +1,56 @@
+#!/bin/bash
+
+# Much of this file (and the entire recipe in fact) has been taken from the
+# root-feedstock.
+
+export STDCXX=17
+export VERBOSE=1
+
+# Manually set the deployment_target
+# May not be very important but nice to do
+OLDVERSIONMACOS='${MACOSX_VERSION}'
+sed -i -e "s@${OLDVERSIONMACOS}@${MACOSX_DEPLOYMENT_TARGET}@g" src/cmake/modules/SetUpMacOS.cmake
+
+declare -a CMAKE_PLATFORM_FLAGS
+if [ "$(uname)" == "Linux" ]; then
+    CMAKE_PLATFORM_FLAGS+=("-DCMAKE_AR=${GCC_AR}")
+    CMAKE_PLATFORM_FLAGS+=("-DCLANG_DEFAULT_LINKER=${LD_GOLD}")
+    CMAKE_PLATFORM_FLAGS+=("-DDEFAULT_SYSROOT=${PREFIX}/${HOST}/sysroot")
+    CMAKE_PLATFORM_FLAGS+=("-DRT_LIBRARY=${PREFIX}/${HOST}/sysroot/usr/lib/librt.so")
+
+    # Fix up CMake for using conda's sysroot
+    # See https://docs.conda.io/projects/conda-build/en/latest/resources/compiler-tools.html?highlight=cmake#an-aside-on-cmake-and-sysroots
+    CMAKE_PLATFORM_FLAGS+=("-DCMAKE_TOOLCHAIN_FILE=${RECIPE_DIR}/cross-linux.cmake")
```

Comment:
This is unnecessary

Reply:
I copied this over from https://github.com/conda-forge/root-feedstock/blob/master/recipe/build.sh#L22. So it should be removed there as well? @chrisburr